localization/en.ftl

```diff
@@ -243,7 +243,7 @@ manage-metadata-desc = Manage Metadata, such as GLesmos or pinned/unpinned statu
 intellisense-name = Intellisense
 intellisense-desc = Brings several common IDE features to Desmos, including autocompletion suggestions, function call help, and jump to definition. Documentation here:
 intellisense-opt-subscriptify-name = Auto-Subscriptify
-intellisense-opt-subscriptify-desc = Automatically add subscripts to variable/function names when they are typed without subscripts.
+intellisense-opt-subscriptify-desc = Automatically predicts text of variables/functions with subscripts when they are typed without subscripts.
```

Comment:
"predicts" sounds like it's doing more than it does.
Maybe "converts names to subscripts"? All it's does is convert `fxyz` to `f_{xyz}` sometimes.

Reply:
I changed the word to "converts". I also updated the Spanish version accordingly.